### DISCUSSION TITLE
Get livewire value from a view

### DISCUSSION THREAD

**Discussion by DevHoracioRodriguez** [livewire/livewire#5761] [1. Help]
Is there a way to get values (array) from livewire when used like the code below shows? Look for: `<livewire:dynamic-controls/>` below.

```
<x-app-layout>
    <x-slot name="header">
        <h2>
            {{ __('Create Form') }}
        </h2>
    </x-slot>

    <div class="py-12">
        <div class="max-w-7xl mx-auto sm:px-6 lg:px-8">
            <div class="bg-white dark:bg-gray-800 shadow sm:rounded-lg">
                <div class="p-6 sm:p-8">
                    <form method="POST" action="{{ route('form.store') }}" class="mt-6 space-y-6">
                        @csrf
                        @method('POST')
                        @include('form.partials.create-form-fields')

                        <livewire:dynamic-controls/>
                        @dd($controls)

                        <div class="flex items-center justify-end mt-4 mb-4">
                            @include('form.partials.create-form-buttons')
                        </div>
                    </form>
                </div>
            </div>
        </div>
    </div>
</x-app-layout>
```

The idea is to get an array from inside <livewire:dynamic-controls/> ($controls) and send it to the form.store. Since this is my first livewire component I thought that was able to have access to it but when I try dd($controls) the variable is undefined. Notice: the component renders fine and works as expected. I just need to save $controls array value along with other form fields. 

Any ideas?

**Comment by onemoreahmad** [comment]
You may have a look at Events, https://laravel-livewire.com/docs/2.x/events
When `$controls` data changes fire an event to its parent to update it.
